.github/workflows/deploy-gcp-tests.yml

```diff
@@ -962,16 +962,80 @@ jobs:
           ${{ inputs.test_id }} | \
           tee --output-error=exit /dev/stderr | \
           grep --max-count=1 --extended-regexp --color=always \
-          -e 'estimated progress.*current_height.*=.*179[0-9][0-9][0-9][0-9].*remaining_sync_blocks' \
+          -e 'estimated progress.*current_height.*=.*180[0-9][0-9][0-9][0-9].*remaining_sync_blocks' \
```

Comment:
This line is actually redundant, 1,800,000 - 1,809,999 is covered by the next regex.

```suggestion
```